### ISSUE TITLE
[bug] Style block inside the document, should updateURLs accordion to base element href first, or not handled

### ISSUE THREAD

**Issue by rhlin** [jhildenbiddle/css-vars-ponyfill#129]
## Description
Style block inside the document, should updateURLs accordion to base element's href first, then if no base element , according to file path. Or, do not handle the url.

## Enviroment
css-vars-ponyfill version: 2.3.1 (the latest version)
browser: IE11 / win10

I'm using css-vars-ponyfill in Angular project. Angular framework use a lot of  inseted inline styles. And I also use hyper link to import styles.  so the option `updateURLs` can't be simplely set to `false`.

## Input

index.html
``` html
<!DOCTYPE html>
<html lang="en-us">
    <head>
        <title>test path</title>
        <base href="/foo/">
        <style>
            :root {
                --my-color: red;
            }
        </style>
        <link rel="stylesheet" href="/foo.css">
        <style>
            h2 {
                background: url('./bar2.jpg') var(--my-color);
            }
        </style>
    </head>
    <body>
        <h1>h1 text</h1>
        <h2>h2 text</h2>
        <script src="/node_modules/css-vars-ponyfill/dist/css-vars-ponyfill.js" type="text/javascript"></script>
        <script defer>
            window.cssVars({ watch: true, silent: true});
        </script>
    </body>
</html>

```

foo.css
``` css
h1 {
    background: url('./bar1.jpg') var(--my-color);
}

```
use `npx  http-server  ./xxx(dir name here)` start up an http server, listenning on http://127.0.0.1:8081

## Expected Output In IE 11
``` html
<!DOCTYPE html>
<html lang="en-us">
    <head>
        <title>test path</title>
        <base href="/foo/">
        <style data-cssvars-job="1" data-cssvars="src">
            :root {
                --my-color: red;
            }
        </style>
        <link disabled="" href="/foo.css" rel="stylesheet" data-cssvars-job="1" data-cssvars="src" data-cssvars-group="1">
        <style data-cssvars-job="1" data-cssvars="out" data-cssvars-group="1">h1{background:url('http://127.0.0.1:8081/bar1.jpg') red;}</style>
        <style disabled="" data-cssvars-job="1" data-cssvars="src" data-cssvars-group="2">
            h2 {
                background: url('./bar2.jpg') var(--my-color);
            }
        </style>
        <style data-cssvars-job="1" data-cssvars="out" data-cssvars-group="2">h2{background:url('http://127.0.0.1:8081/foo/bar2.jpg') red;}</style>
    </head>
    <body>
        <h1>h1 text</h1>
        <h2>h2 text</h2>
        <script src="/node_modules/css-vars-ponyfill/dist/css-vars-ponyfill.js" type="text/javascript"></script>
        <script defer="">
            window.cssVars({ watch: true, silent: true});
        </script>
    
    </body>
</html>
```

## Actual Ouput in IE11
``` html
<!DOCTYPE html>
<html lang="en-us">
    <head>
        <title>test path</title>
        <base href="/foo/">
        <style data-cssvars-job="1" data-cssvars="src">
            :root {
                --my-color: red;
            }
        </style>
        <link disabled="" href="/foo.css" rel="stylesheet" data-cssvars-job="1" data-cssvars="src" data-cssvars-group="1"><style data-cssvars-job="1" data-cssvars="out" data-cssvars-group="1">h1{background:url('http://127.0.0.1:8081/bar1.jpg') red;}</style>
        <style disabled="" data-cssvars-job="1" data-cssvars="src" data-cssvars-group="2">
            h2 {
                background: url('./bar2.jpg') var(--my-color);
            }
        </style><style data-cssvars-job="1" data-cssvars="out" data-cssvars-group="2">h2{background:url('http://127.0.0.1:8081/bar2.jpg') red;}</style>
    </head>
    <body>
        <h1>h1 text</h1>
        <h2>h2 text</h2>
        <script src="/node_modules/css-vars-ponyfill/dist/css-vars-ponyfill.js" type="text/javascript"></script>
        <script defer="">
            window.cssVars({ watch: true, silent: true});
        </script>
    
    </body>
</html>
```
In the case: 
h1 background url is updated right，while h2 background wrong。
h2 background url should use the href property value of base element as url prefix.
`h2{background:url('http://127.0.0.1:8081/bar2.jpg') red;}` is supposed to be `h2{background:url('http://127.0.0.1:8081/foo/bar2.jpg') red;}`

[Document](https://jhildenbiddle.github.io/css-vars-ponyfill/#/?id=updateurls) says `updateURLs: 
 When true, the ponyfill will parse each block of external CSS for relative url() paths and convert them to absolute URLs. `  
To my understading , the ponyfill will parse **EXTERNAL** CSS **only**. Style block inside the document should not be handled.

**Comment by jhildenbiddle** [comment]
Thank for filing the issue, @rhlin.

I will update the behavior of the `updateURLs` options so that only URLs from external stylesheets will be updated. Given your examples above, the output from the ponyfill will be as follows:

```css
h1 {
  background:url('http://127.0.0.1:8081/bar1.jpg') red;
}
```
```css
h2 {
  background:url('./bar2.jpg') red;
}
```

**Comment by jhildenbiddle** [comment]
Bug has been fixed and checked in. I will publish a new version with the fix this weekend.

Thanks again, @rhlin!

**Comment by jhildenbiddle** [comment]
Fix available in 2.3.2.

**Comment by rhlin** [comment]
Now it works, thanks !
